scripts/static-analyser-check.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
+set -exo pipefail
```

Comment:
https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html

* `set -e` exit immediately if any command line fails.
* `set -x` print the command to stderr and then execute it (helps know which commands fail and where).
* `set -o pipefail` exit script when any command in a piped sequence fails.  Example: `a | b | c` if command `a` or `b` fails the script would not exit unless this option is set.